include/ibamr/IBFESurfaceMethod.h

```diff
@@ -547,6 +613,10 @@ class IBFESurfaceMethod : public IBStrategy
     std::vector<libMesh::Order> d_default_quad_order;
     bool d_use_consistent_mass_matrix;
     bool d_use_direct_forcing;
+    double d_mu;
+    double d_wss_calc_width, d_p_calc_width;
+    double d_traction_activation_time;
+    bool d_use_l2_lagrange_family;
```

Comment:
It would be more generalizable to just provide the `FEFamily` and `FEOrder` values instead of using booleans.